app/code/Magento/Cookie/etc/adminhtml/system.xml

```diff
@@ -22,14 +22,18 @@
                     <label>Cookie Domain</label>
                     <backend_model>Magento\Cookie\Model\Config\Backend\Domain</backend_model>
                 </field>
-                <field id="cookie_httponly" translate="label" type="select" sortOrder="40" showInDefault="1" showInWebsite="1" showInStore="1">
+                <field id="cookie_max" translate="label" type="text" sortOrder="40" showInDefault="1" showInWebsite="1" showInStore="0">
```

Comment:
This setting is most likely to be set just once, so there is not really a reason to have it in the admin. Consider implementing this as a configuration file.